Test2/Pages/SettingPage.cs

```diff
@@ -0,0 +1,104 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using System;
+using System.Collections.Generic;
+
+namespace Test2.Pages
+{
+    internal class SettingPage
+    {
+        private IWebDriver driver;
+        public SettingPage(IWebDriver webDriver)
+        {
+            driver = webDriver;
+        }
+
+        public IWebElement DateFormat => driver.FindElement(By.XPath("//th[text()='Date format:']/..//select[@id='dateFormat']"));
+        public IWebElement DefaultCurrency => driver.FindElement(By.XPath("//th[text()='Default currency:']/..//select[@id='currency']"));
+        public IWebElement NumberFormat => driver.FindElement(By.XPath("//th[text()='Number format:']/..//select[@id='numberFormat']"));
+        public IWebElement ButtonSave => driver.FindElement(By.Id("save"));
+        public IWebElement LogutButton => driver.FindElement(By.XPath("//div[contains(@class, 'login')]"));
+        public IWebElement simb => driver.FindElement(By.Id("currency"));
+
+        public string EndDay => driver.FindElement(By.XPath("//th[text()='End Date: *']/..//input")).GetAttribute("value");
+
+        private void WaitForReady()
+        {
+            try
+            {
+                WebDriverWait wait = new WebDriverWait(driver, TimeSpan.FromSeconds(2));
+                wait.Until(ExpectedConditions.ElementToBeClickable(ButtonSave));
+            }
+            catch (Exception ex)
+            {
+            }
+        }
+        private void WaitForAlert()
+        {
+            new WebDriverWait(driver, TimeSpan.FromSeconds(10))
+             .Until(ExpectedConditions.AlertIsPresent());
+            IAlert alert = driver.SwitchTo().Alert();
+            alert.Accept();
+        }
+        public void Сonfigure(string format)
```

Comment:
1. Please add an empty line between methods.
2. Name should be more descriptive.

```suggestion
        }
        
        public void SetDateFormat(string format)
```